docs/code-of-conduct.rst

```diff
@@ -48,7 +64,9 @@ This code of conduct applies to all spaces managed by Write the Docs. This inclu
 * meetup.com discussion boards
 * Any other forums created by the which the community uses for communication.
```

Comment:
I would explicitely mention Slack and Twitter hashtag and mentions in separate bullets

Reply:
Slack and twitter already have their own bullets, but I'll specifically mention mentions too.